README.md

```diff
@@ -128,86 +91,24 @@ You can dynamically decide which transport to use, basing e.g. on the user-agent
-and set a new session cookie), you need to use the `setSession` directive:
-
-````scala
-import akka.http.scaladsl.server.Directives._
-
-import com.softwaremill.session.SessionDirectives._
-import com.softwaremill.session.SessionOptions._
-
-path("login") {
-  post {
-    entity(as[String]) { body =>
-      setSession(oneOff, usingCookies, 812832L) { ctx =>
-        ctx.complete("ok")
-      }
-    }
-  }
-}
-````
+and set a new session cookie), you need to use the `setSession` directive. See how it's done in [scala](https://github.com/softwaremill/akka-http-session/blob/master/example/src/main/scala/com/softwaremill/example/session/SetSessionScala.scala) and [java](https://github.com/softwaremill/akka-http-session/blob/master/example/src/main/java/com/softwaremill/example/session/SetSessionJava.java).
 
 Note that when using cookies, their size is limited to 4KB, so you shouldn't put too much data in there (the signature 
 takes about 50 characters). 
 
-You can require a session to be present, optionally require a session or get a full description of possible session 
-decode outcomes:
-
-````scala
-path("secret") {
-  get {
-    requiredSession(oneOff, usingCookies) { session => // type: Long, or whatever the T parameter is
-      complete { "treasure" }
-    }
-  }
-} ~
-path("open") {
-  get {
-    optionalSession(oneOff, usingCookies) { session => // type: Option[Long] (Option[T])
-      complete { "small treasure" }
-    }
-  }
-} ~
-path("detail") {
-  get {
-    // type: SessionResult[Long] (SessionResult[T])
-    // which can be: Decoded, CreatedFromToken, Expired, Corrupt, NoSession
-    session(oneOff, usingCookies) { session => 
-      complete { "small treasure" }
-    }
-  }
-}
-````
-
-If a required session is not present, by default a 403 is returned. Finally, a session can be invalidated:
+You can require a session to be present, optionally require a session or get a full description of possible session decode outcomes. See examples for [java](https://github.com/softwaremill/akka-http-session/blob/master/example/src/main/java/com/softwaremill/example/session/VariousSessionsJava.java) and [scala](https://github.com/softwaremill/akka-http-session/blob/master/example/src/main/scala/com/softwaremill/example/session/VariousSessionsScala.scala).
 
-````scala
-path("logout") {
-  get {
-    requiredSession(oneOff, usingCookies) { session => 
-      invalidateSession(oneOff, usingCookies) {
-        complete { "logged out" }
-      }
-    }
-  }
-}
-````
+If a required session is not present, by default a `403` HTTP status code is returned. Finally, a session can be invalidated. See how it's done in See examples for [java](https://github.com/softwaremill/akka-http-session/blob/master/example/src/main/java/com/softwaremill/example/session/SessionInvalidationJava.java) and [scala](https://github.com/softwaremill/akka-http-session/blob/master/example/src/main/scala/com/softwaremill/example/session/SessionInvalidationScala.scala).
```

Comment:
`See how it's done in See examples` - the second `See` is redundant here I guess :)